### PR TITLE
New admin buttens for Reece

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Just some harmless butters for admins to boop. Requested by Reece

[why]
Reece the Foooool asked me to make a few admin buttens - A failer of a coder

### CODE REVIEW

**Comment by geemiesif** [comment]
this is for the goof foam event but with semen or fem cum 
+1

File: code/modules/events/vent_clog.dm [hunk 1]
```diff
@@ -79,12 +79,28 @@
...
+/datum/round_event_control/vent_clog/female
+	name = "FemCum stationwide"
+	typepath = /datum/round_event/vent_clog/female
+	max_occurrences = 0
+
+/datum/round_event/vent_clog/female
+	reagentsAmount = 100
+
+/datum/round_event_control/vent_clog/male
+	name = "Semen stationwide"
```
Comment (Ragolution) [inline code/modules/events/vent_clog.dm]: name = "Scrubber Bukake"

Comment (Ragolution) [inline code/modules/events/vent_clog.dm:99]:
```suggestion
	name = "Scrubber Bukkake"
```

File: code/modules/events/vent_clog.dm [hunk 2]
```diff
@@ -79,12 +79,28 @@
...
+	reagentsAmount = 100
+
 /datum/round_event_control/vent_clog/plasma_decon
 	name = "Plasma decontamination"
 	typepath = /datum/round_event/vent_clog/plasma_decon
 	max_occurrences = 0
 
-/datum/round_event/vent_clog/beer
+/datum/round_event_control/vent_clog/female
+	name = "FemCum stationwide"
```
Comment (Ragolution) [inline code/modules/events/vent_clog.dm:91]:
```suggestion
	name = "Scrubber FemCum Bukkake"
```

**Review by nik707 — APPROVED** [review]
im doing this for reece

**Pull request merged**